project.properties

```diff
@@ -6,4 +6,4 @@
 # The Java project include this file as a Gradle properties
 # And the Dotnet projects include and parse this file.
 dafnyVersion=4.2.0
-dafnyVerifyVersion=4.2.0
+dafnyVerifyVersion=4.7.0
```

Comment:
Just FYI, I've had to add support to smithy-dafny to generate these project.properties files in order for the nightly builds to be able to use a newer Dafny everywhere. It isn't yet outputting this property, so that means the nightly builds are no longer verifying with the latest Dafny master.

The good news is this change helps close that gap, but it still exists and we should track it somehow.